src/views/Main.js

```diff
@@ -6,48 +6,77 @@
  * @flow
  */
 
-import React, { useState, useEffect } from 'react';
+import React, {useState, useEffect} from 'react';
 import useNFC from '../utils/nfc';
```

Comment:
unused import?

Reply:
yeah there’s a few inconsistencies, I quickly merged stanfords changes into this and missed some stuff. 

@kozr could you clean it up before merge?